### DISCUSSION TITLE
环境变量写ACCESS_TOKEN的时候超出4KB大小限制，以前的方法也不能用了

### DISCUSSION THREAD

**Discussion by UptonEdward** [spencerwooo/onedrive-vercel-index#244] [Q&A] [answered]
在以前Closed的Issue里找到了4KB大小限制的对应措施，说要修改/pages/api.ts的15-20行。但是现在api.ts已经大改了，目前该怎么解决这个4KB问题呢？

**Comment by spencerwooo** [comment, marked as answer]
您完全没看新版本的 announcement：

> Starting now, you don't have to manually acquire your tokens, and you don't need to register a client at Microsoft anymore (PR #210 and Discussion #215).

和最新 deployment 文档：

> https://github.com/spencerwooo/onedrive-vercel-index#new-users

吗？
